CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.35.0] - 2022-02-17
+
+### Added
+- seek preview is now configurable via `isSeekPreviewEnabled` from SeekBarConfig or UIConfig
```

Comment:
Slight nitpick to aviod confusion about the config options. We should mention the respective property names
```suggestion
- Seek preview is now configurable via `UIConfig.enableSeekPreview` or `SeekBarConfig.isSeekPreviewEnabled`
```